onert-micro/luci-interpreter/src/kernels/MISOKernel.h

```diff
@@ -57,12 +64,19 @@ class MISOKernel
   const circle::Tensor *input2() const { return _input2_tensor; }
   const circle::Tensor *input3() const { return _input3_tensor; }
 
+  const circle::Tensor *input4() const
+  {
+    assert(_input4_tensor != nullptr);
+    return _input4_tensor;
+  }
+
   const circle::Tensor *output() const { return _output_tensor; }
 
 private:
   const circle::Tensor *_input1_tensor;
   const circle::Tensor *_input2_tensor;
   const circle::Tensor *_input3_tensor;
+  const circle::Tensor *_input4_tensor; // optional
```

Comment:
Add optional field for tensor to support StridedSlice and use MISOKernel helper for this kernel